### ISSUE TITLE
Cannot force 144hz unplugged whatever I do.

### ISSUE THREAD

**Issue by PetarCMD** [seerge/g-helper#3458]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

My laptop is TUF A15 ryzen 7 7435hs 4050 rtx with Windows 11 24h2 (recently switched to 23h2 still same issue)

When I unplug my charger, the refresh rate changes from 144hz to 60hz. Things to note:

- the screen doesn't turn black in order for the refresh rate to change, it instantly does.

- I have tried every setting in G-Helper, but nothing seems to change my refresh rate while unplugged.

- My CPU doesn't have a iGPU, so no GPU settings in G-Helper.

- i've tried installing nvidia drivers directly from the ASUS webpage, doesn't work

- newest nvidia driver doesn't work (each time nvidia driver is uninstalled with DDU and clean settings)

- i've checked every Armory Crate option, nothing helped

- uninstalled Armory Crate with the official uninstaller, currently have GHelper, using 144hz + OD, still 60hz when unplugged

- tried all display properties in the windows menu (dynamic; 144hz) still the same.

### How to reproduce the bug?

1. Plug the laptop and the screen is 144hz
2. Unplug the charger - instantly changes to 60hz (might be 72hz)
3. If while unplugged I manually set it to 60hz the screen blacks out and after 1 sec it is 60hz again
4. No settings in G-Helper have managed to help me

### Logs

If necessary I will provide.

### Device and Model

ASUS TUF A15 ryzen 7 7435hs 4050 rtx with Windows 11 24h2 (23h2)

### Armoury Crate

Uninstalled

### Asus Services

None, all stopped by G-Helper

### Version

0.199

### OS

Windows 24h2 (23h2)

**Comment by seerge** [comment]
@PetarCMD hello,

How is this related to G-Helper ? :) Unless you have `Auto` refresh rate selected in the app or manually click refresh rate button, it won't touch your refresh rates. 

Did you uninstall Asus stuff, especially **Asus Smart Display Control ?**

Can you select 144Hz in windows settings when on battery ?

**Comment by PetarCMD** [comment]
- I uninstalled everything Asus related.
- Yes I can select 144Hz in windows settings when on battery.
Sometimes if I open an app or, for example, a youtube video, it is 144Hz while it is opened. Immediately after you close it - again 60Hz.
My guess is that it is something related to Nvidia drivers + Windows settings not able to communicate properly. It is not related to G-Helper, I just had to double check. Thank you for the answers and of course, if you have any other suggestions I will try them.

**Comment by seerge** [comment]
@PetarCMD hello,

If refresh rate changes by itself when you launch app or video, it means something else is changing it.

G-Helper can change refresh rate via windows API (i.e. same thing as you do in settings manually). For a sanity check you can open app log at `%appdata%\GHelper\` and verify that it's not G-Helper changing it or just quit app and see if it still happens.

Try to open windows Task Manager and search for "Asus" processes there. Do you see anything running ?

**Comment by PetarCMD** [comment]
Hello Seerge,
G-Helper isn't doing anything with the refresh rate and also no Asus processes in TM. Downgraded to 23h2 with Nvidia drivers from the Asus website and it stays 144hz unplugged ONLY if it's set in Windows to Dynamic refresh rate (i find that funny). If I manually set it to 144hz and 144hz only, it drops down. If I install the newest driver from Nvidia directly, it never stays 144hz no matter what option I choose in windows.
I'm currently Win11 24h2 and dynamic refresh rate is greyed out, which is the cherry on the cake.
Anyways, thanks again for the support, had to check if it was due to G-Helper and maybe you had similar problems, but I guess i'm the odd one. Haha
